.github/workflows/ci-dev-ops.yml

```diff
@@ -26,17 +25,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Install yarn
-        run: |
-          corepack prepare yarn@stable --activate
-          npm install -g yarn
-
       - name: Install Node
         uses: actions/setup-node@v4
         with:
           node-version: 20
           cache: "yarn"
 
+      - name: Install yarn
+        run: corepack prepare yarn@stable --activate
+
```

Comment:
hi @jplodge-pro - this doesn't work locally with act for some reason.  i was only able to install yarn as per the below approach.  is it causing some issues?

```
      - name: Install yarn
        run: |
          corepack prepare yarn@stable --activate
          npm install -g yarn

```

Reply:
Ah that's what I was trying to remember! No worries. I will revert. 
From my reading of the yarn install docs, only the `corepack` command is required. But  empirical evidence wins out.